dir.props

```diff
@@ -472,9 +472,8 @@
   <PropertyGroup>
     <TestTFM Condition="'$(TestTFM)'=='' AND '$(TargetGroup)'=='netstandard1.7'">netcoreapp1.1</TestTFM>
     <TestTFM Condition="'$(TestTFM)'==''">netcoreapp1.0</TestTFM>
-    <FilterToTestTFM Condition="'$(FilterToTestTFM)'=='' AND '$(TestTFM)'=='netcoreapp1.1'">netcoreapp1.1</FilterToTestTFM>
-    <!-- we default FilterToTestTFM to netcoreapp1.0 if it is not explicity defined -->
-    <FilterToTestTFM Condition="'$(FilterToTestTFM)'==''">netcoreapp1.0</FilterToTestTFM>        
+    <!-- we default FilterToTestTFM to netcoreapp1.1 if it is not explicity defined -->
+    <FilterToTestTFM Condition="'$(FilterToTestTFM)'==''">netcoreapp1.1</FilterToTestTFM>        
```

Comment:
FiterToTestTFM is today only set in here or when you call it via Helix and you want to test for different TFMs like net46 for example. What we are doing here, is that if that is not passed (by Helix) we assume that we are testing for netcoreapp1.1

If TestTFM is not provided we will continue to assume that it is netcoreapp1.0, because not all of the test projects today are targeting netcoreapp1.1 yet. Once they all support it, then we will be able to change the default TestTFM to be netcoreapp1.1.